definitions/npm/@react-navigation/core_v3.x.x/flow_v0.60.x-/core_v3.x.x.js

```diff
@@ -720,6 +711,16 @@ declare module '@react-navigation/core' {
     config?: NavigationTabRouterConfig
   ): NavigationRouter<*, *>;
 
+  declare type _SafeAreaViewForceInsetValue = 'always' | 'never' | number;
+  declare type _SafeAreaViewInsets = $Shape<{|
```

Comment:
Not sure I get it, or maybe I wasn't clear enough in the previous review.   `$Shape<{|`   
Exact objects should be preferred as default, but if some cases really needs to be more relaxed, then a normal object will be fine.